microcosm_resourcesync/tests/endpoints/test_http_endpoint.py

```diff
@@ -143,9 +143,6 @@ def test_read_follow_page(self):
                         self=dict(
                             href="http://example.com/api/foo?offset=2&limit2",
                         ),
-                        prev=dict(
```

Comment:
Why?

Reply:
Because it tries to follow that link, since it's distinct from the :code:`/api` link. However, the mock does not have another answer. The test used to pass "accidentally", because the mock raised a `StopIteration` which caused the iterator to be done. This has always been a problem, and was fixed in 3.7.